patch.py

```diff
@@ -0,0 +1,19 @@
+#!/bin/python3.6
+import sys
+
+patch=open(sys.argv[1], "r").readlines()
+orig=open(sys.argv[2],"r")
+origlines = orig.readlines()
+
+for idx,oline in enumerate(origlines):
+  subst = oline.split('=')
+  for idx1,pline in enumerate(patch):
+    if subst[0] in pline:
```

Comment:
I would suggest to make this test more exact, to be sure it does not "fuzzy" match the wrong string

Reply:
like this?

Reply:
I think that should work